FFCDemoPaymentService.Tests/TestResults.info

```diff
@@ -0,0 +1,285 @@
+SF:/c/Repos/ffc-demo-payment-service-core/FFCDemoPaymentService/Program.cs
```

Comment:
Add this file to .gitignore as we don't need it in the repo.  I'm not sure how configurable it is but it would make life easier if it was output to the TestResults folder too.

Reply:
This was a duplicate of the file in the TestResults folder so removed.